Packs/AzureStorageContainer/Integrations/AzureStorageContainer/AzureStorageContainer.py

```diff
@@ -0,0 +1,921 @@
+
+
+def list_blobs_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    List Blobs under the specified container.
+
+    Args:
+        client (Client): Azure Blob Storage API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+
+    """
+    container_name = args['container_name']
+    limit = args.get('limit') or '50'
+    prefix = args.get('prefix')
+    page = arg_to_number(args.get('page') or '1')
+
+    marker = ''
+    readable_message = f'{container_name} Container Blobs List:\n Current page size: {limit}\n ' \
+                       f'Showing page {page} out others that may exist'
+    if page > 1:  # type: ignore
+        marker = get_pagination_next_element(limit=limit, page=page,  # type: ignore
+                                             client_request=client.list_blobs_request,
+                                             params={"container_name": container_name,
+                                                     "prefix": prefix})  # type: ignore
+
+        if not marker:
+            return CommandResults(
+                readable_output=readable_message,
+                outputs_prefix='AzureStorageContainer.Container',
+                outputs=[],
+                raw_response=[]
+            )
+
+    response = client.list_blobs_request(container_name, limit, prefix, marker)
+
+    tree = ET.ElementTree(ET.fromstring(response))
+    root = tree.getroot()
+
+    raw_response = []
+    blobs = []
+
+    for element in root.iter('Blob'):
+        data = {'name': element.findtext('Name')}
+        blobs.append(dict(data))
+        properties = {}
+        for blob_property in element.findall('Properties'):
+            for attribute in blob_property:
+                properties[attribute.tag] = attribute.text
+
+        data['Property'] = properties  # type: ignore
+        raw_response.append(data)
+
+    outputs = {"name": container_name, "Blob": blobs}
+    readable_output = tableToMarkdown(
+        readable_message,
+        outputs.get('Blob'),
+        headers='name',
+        headerTransform=string_to_table_header
+    )
+
+    command_results = CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='AzureStorageContainer.Container',
+        outputs_key_field='name',
+        outputs=outputs,
+        raw_response=raw_response
+
+    )
+
+    return command_results
+
+
+def create_blob_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Create a new Blob under the specified Container.
+
+    Args:
+        client (Client): Azure Blob Storage API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+
+    """
+    container_name = args['container_name']
+    file_entry_id = args['file_entry_id']
+    blob_name = args.get('blob_name')
```

Comment:
mandatory args should be accessed with `[]` and not `get`, as with the two above

Reply:
This is not a mandatory argument.
Because the blob is created using XSOAR File  Entry ID, if no name is provided, the name of the blob will be the same as the name of the existing file in XSOAR.
This is indicated in the description of the argument.